index.js

```diff
@@ -24,18 +27,33 @@ cmd
 .option('-p, --port <n>', 'Port to start the HTTP server', parseInt)
 .parse(process.argv)
 
+var options = {
+  key: fs.readFileSync('./certs/localhost.key'),
+  cert: fs.readFileSync('./certs/localhost.crt')
+}
+
 // Launch server with web sockets
-var server = http.createServer(app)
+var server = http.createServer(options, app)
 var broker = new mosca.Server({})
 broker.attachHttpServer(server)
 
 process.env.PORT = cmd.port || process.env.PORT
-process.env.LOCAL_URL = 'http://localhost:' + process.env.PORT
+process.env.LOCAL_URL = 'https://localhost:' + process.env.PORT
+process.env.REDIR_PORT = (parseInt(process.env.PORT) + 1).toString()
+
+// Launch a http common server that redirects to https
+redir
+/*
+.createServer() */
+.get('*', function (req, res) {
+  res.redirect(process.env.LOCAL_URL)
+})
+.listen(process.env.REDIR_PORT, function () {
+  console.log('Redirection server listening on ' + process.env.REDIR_PORT)
+})
 
 server.listen(process.env.PORT, function () {
-  console.log('👾  Netbeast dashboard started on %s:%s',
-  server.address().address,
-  server.address().port)
+  console.log('👾  Netbeast dashboard started on ', process.env.LOCAL_URL)
```

Comment:
Seems alright but LOCAL_URL should be always the same as the result on server.address(), so I prefer to leave it different to check it is working properly.